src/Pages/ProfilePage/ProfilePage.jsx

```diff
@@ -0,0 +1,165 @@
+import axios from "axios";
+import { EditProfileModal, UserPosts } from "Components";
+import { useAuthContext, useReducerContext, useUserContext } from "Context";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { FETCH_POSTS } from "Utils/Action";
+import * as styles from "./ProfilePage.module.css";
```

Comment:
I think, there is no need to do it like this. You can simply use `import styles from "..."` and it will work also.

Reply:
Thanks for this I will try it